scripts/dockerrun.sh

```diff
@@ -129,5 +129,8 @@ docker run $INTERACTIVE -t --rm --sig-proxy=true \
     -e STORAGE_KEY \
     -e STORAGE_ACCOUNT \
     -e STORAGE_CONTAINER \
```

Comment:
These names are somewhat ambiguous given the new checksum variables.  Would it make sense to qualify them as for what they are intended to be used for (e.g. artifacts)?